doc/devel/testing.rst

```diff
@@ -40,11 +47,7 @@ particularly useful ones are:
 ``-v`` or ``--verbose``        Be more verbose
 ``-n NUM``                     Run tests in parallel over NUM
                                processes (requires pytest-xdist_)
-``--timeout=SECONDS``          Set timeout for results from each test
-                               process (requires pytest-timeout_)
```

Comment:
Does anybody use `--timeout` explicitly when running the tests? I don't and I don't see why one should.